docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -544,36 +629,37 @@ This command will create an Enterprise Application (so a Service Principal) with
+
 ```powershell
-$subscriptionID = "<Your Azure Subscription ID>" ## Your Azure Subscription
-$principalId = "21733e48-ab53-4ca9-ac40-1a92daa528ac" ## Principal Id after creating system identity for Backend Processor Container app 
+$subscriptionID = "<Your Azure Subscription ID>" ## Your Azure Subscription id which you can find on the azure portal
+$principalId = "<your principal id which was generated above>" ## Principal Id after creating system identity for Backend Processor Container app 
 $roleNameOrId =  "Azure Service Bus Data Receiver" ## Built in role name
-$namespaceName = "taskstracker" ##Name of your Service Bus Namespace
+
 
 az role assignment create `
 --assignee $principalId `
 --role $roleNameOrId `
---scope /subscriptions/$subscriptionID/resourcegroups/$RESOURCE_GROUP/providers/Microsoft.ServiceBus/namespaces/$namespaceName
+--scope /subscriptions/$subscriptionID/resourcegroups/$RESOURCE_GROUP/providers/Microsoft.ServiceBus/namespaces/$NamespaceName
 ```
 
 ##### 3. Grant Backend API App the Azure Service Bus Data Sender Role
 We'll do the same with Backend API container app, but we will use a different Azure built-in roles for Azure Service Bus which is the role `Azure Service Bus Data Sender` as the Backend API is a publisher of the messages. Run the command below to associate the `system-assigned` with access-control role `Azure Service Bus Data Sender`:
 
 ```powershell
 $subscriptionID = "<Your Azure Subscription ID>" ## Your Azure Subscription
-$principalId = "774aaaab-e3a9-452c-a168-54dcc88a8500" ## Principal Id after creating system identity for Backend API Container app 
+$principalId = "<your principal id which was generated in module 4. You can find it on the azure portal under the specific container identity section>" ## Principal Id after creating system identity for Backend API Container app
 $roleNameOrId =  "Azure Service Bus Data Sender" ## Built in role name
-$namespaceName = "taskstracker" ##Name of your Service Bus Namespace
+
 
 az role assignment create `
 --assignee $principalId `
 --role $roleNameOrId `
---scope /subscriptions/$subscriptionID/resourcegroups/$RESOURCE_GROUP/providers/Microsoft.ServiceBus/namespaces/$namespaceName 
+--scope /subscriptions/$subscriptionID/resourcegroups/$RESOURCE_GROUP/providers/Microsoft.ServiceBus/namespaces/$NamespaceName
 ```
 
 ##### 4. Restart container apps
-Lastly, we need to restart both container apps revision, to do so run the command below, you need to do the same command for the Backend API App too.
+Lastly, we need to restart both container apps revisions to ensure that the identity changes are properly reflected. To do so run the commands below:
```

Comment:
@waelkdouh to confirm, you checked without restart, and it didn't work, right? a restart is needed to pick up the role assignment correctly?